conf/defaults.ini

```diff
@@ -363,6 +363,7 @@ enabled = false
 config_file = /etc/grafana/ldap.toml
 allow_sign_up = true
 sync_cron = @hourly
+active_sync_enabled = false
```

Comment:
Move this own to just below the other sync setting. If someone looks at defaults.ini that makes it possible to understand that this is enterprise only.